src/common/version.ts

```diff
@@ -49,15 +53,42 @@ export class Versions {
         return Versions.getValue(Constants.versionTempSensor, "1.0") as string;
     }
 
+    public static updateSystemModuleImageVersion(templateJson: any, moduleName: string, versionMap: Map<string, string>) {
+        if (templateJson) {
+            const sysModuleImage =
+                templateJson.modulesContent.$edgeAgent["properties.desired"].systemModules[moduleName].settings.image;
+            templateJson.modulesContent.$edgeAgent["properties.desired"].systemModules[moduleName].settings.image =
+                Versions.getNewImageVersionJson(sysModuleImage, versionMap);
+        }
+    }
+
+    public static edgeHubVersion(): string {
+        return Versions.getDefaultEdgeRuntimeVersion();
```

Comment:
this resolves to 1.0 always, correct? Shouldn't it resolve to 1.1 or 1.0 depending on the user selected Edge runtime?

Reply:
when the user selects a new runtime version, it becomes the new default version, the workspace config is updated and getDefaultEdgeRuntimeVersion will return whatever the latest default version is selected. I call it default version because after it has been selected, all new modules will be generated using this version.

---
In reply to: [602606639](https://github.com/microsoft/vscode-azure-iot-edge/pull/569#discussion_r602606639) [](ancestors = 602606639)